react-effects/my-app/src/List.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import readItems, { Item } from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState<Item[]>([]);
+  const [error, setError] = useState<unknown>();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+  useEffect(() => {
+    async function getItems() {
+      try {
+        const items = await readItems();
+        setIsLoading(false);
+        setItems(items);
+      } catch (error) {
+        setIsLoading(false);
```

Comment:
Good. The easiest way to make sure isLoading gets set to false is to do it in the `finally` clause.